lib/CodeGen/CGExpr.cpp

```diff
@@ -3664,13 +3665,16 @@ LValue CodeGenFunction::EmitCastLValue(const CastExpr *E) {
     return MakeNaturalAlignAddrLValue(EmitDynamicCast(V, DCE), E->getType());
   }
 
+  case CK_DynamicPtrBounds:
+  case CK_AssumePtrBounds:
```

Comment:
This issue is same as below for EmitDynamicBoundsCast method
For AssumePtrBounds, it only generaets explicit type casting operation not dynamic bounds check